google/cloud/forseti/services/actions/action_engine.proto

```diff
@@ -17,13 +17,13 @@ syntax = "proto3";
 import "google/protobuf/timestamp.proto";
 
 service ActionEngine {
-  rpc Ping(PingRequest) returns (PingReply) {
+  rpc ping(PingRequest) returns (PingReply) {
```

Comment:
We should be using CamelCase for the RPC endpoints per convention. https://developers.google.com/protocol-buffers/docs/style. I know we went back and forth in group discussions, but I think we need to change these all back. The other updates in this PR look good.